docs/src/what_is_aleph.md

```diff
@@ -0,0 +1,207 @@
+- Still, no matter how unlikely such attacks might be, we take them very seriously and plan to add randomness to Rush in one of the future releases. We decided to go for a version without randomness first, as it gives an incredibly simple and at the same time secure and robust BFT consensus protocol. Adding randomness introduces some complexity into the protocol, so it makes sense to add it on top of a well-tested, working product. The API of the protocol will not change and we will make the use of randomness configurable.
+
+### 2.5 Alerts -- Dealing with Fork Spam.
+We note that the `OrderData` algorithm as described in previous subsections **is resistant to forks**, meaning that as long as there are at most `f` forking nodes, then the resulting consensus protocol is still safe.
+That being said, there are **practical** issues when it comes to dealing with huge amounts of forks. These are extensively described in the [paper](https://arxiv.org/abs/1908.05156) in the section on a  "Fork Bomb Attack" -- we encourage the reader to take a look. We describe here a mitigation to this issue that is implemented in Rush. It is based on the idea of "Alerts", described in the paper, but significantly simplified and made viable for implementation.
+
+The main idea here is that we cannot afford to store an arbitrary amount of units that are forks -- most of them are useless, but without adding a specific mechanism to the protocol, there is no way to discard them. That's why we introduce **Fork Alerts**. A fork alert is a message
+
+**`ForkAlert(sender, forker, proof, units)`**
+
+where `sender` is the node index of the sender, `forker` is the node index of a forking node, `proof` is a pair of units created by forker that constitute a fork and `units` is simply some list of forker's units. Such a message is brodcast by `sender` to all other nodes:
+1. The broadcast is performed using a **Reliable Broadcast** primitive. Roughly speaking, this means that the node does not simply multicast this message to all the nodes but runs a specialized protocol that guarantees that all the nodes receive this message (it the sender is honest) and receive the exact same version of this message (even if the sender is **dishonest**). We give more details on the Rush's implementation of reliable broadcast in the dedicated section.
+2. The intuitive meaning of such an alert is that: "I, `sender`, am aware that `forker` forked and I attach a `proof` of that. Moreover, before I realized this fact, I accepted the following list of units to my local copy of the Dag. Please accept these units as well."
```

Comment:
Please, specify that all units in the list are created by the forker. It is said earlier that "`units` is simply some list of forker's units", but saying it one more time is a good redundancy which minimizes probablity of a misunderstanding

Reply:
Ok, I made this more specific.